frontend/src/components/ContributionMessages/ContributionMessagesFormular.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="contribution-messages-formular">
+    <div>
+      <b-form @submit.prevent="onSubmit" @reset="onReset">
+        <b-form-textarea
+          id="textarea"
+          v-model="form.text"
+          placeholder="Enter something..."
+          rows="3"
+          max-rows="6"
+        ></b-form-textarea>
+        <b-row class="mt-4 mb-6">
+          <b-col>
+            <b-button type="reset" variant="danger">{{ $t('form.reset') }}</b-button>
```

Comment:
```suggestion
            <b-button type="reset" variant="danger">{{ $t('form.cancel') }}</b-button>
```

Like on `ContributionForm`?